packages/p2p/src/components/verification/verification.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Icon, Checklist } from '@deriv/components';
+import Dp2pContext from 'Components/context/dp2p-context';
+import { Localize, localize } from 'Components/i18next';
+import './verification.scss';
+
+const Verification = ({ poi_status }) => {
+    const { nickname, toggleNicknamePopup, is_advertiser } = React.useContext(Dp2pContext);
+
+    const items = [
+        {
+            content: nickname ? (
+                <Localize i18n_default_text='Nickname: {{nickname}}' values={{ nickname }} />
+            ) : (
+                <Localize i18n_default_text='Choose your nickname' />
+            ),
+            status: nickname ? 'done' : 'action',
+            onClick: nickname ? () => {} : toggleNicknamePopup,
+        },
+        {
+            content: (
+                <Localize
+                    i18n_default_text='We’ll need you to upload your documents to verify 
+            your identity'
+                />
+            ),
+            status: poi_status === 'verified' ? 'done' : 'action',
+            onClick: () => (window.location.href = '/account/proof-of-identity'),
```

Comment:
We should consider p2p is a standalone package, this URL should be supplied by the consuming app.

Reply:
ahh, right. Thanks 👍